packages/ses/docs/lockdown.md

```diff
@@ -20,24 +20,38 @@ compatibility vs better tool compatibility.
-| `errorTaming`      | `'safe'`         | `'unsafe'`     | `errorInstance.stack`      |
-| `errorTrapping`    | `'platform'`     | `'exit'` `'abort'` `'report'` `'none'` | handling of uncaught exceptions |
-| `unhandledRejectionTrapping` | `'report'` | `'none'`   | handling of finalized unhandled rejections |
-| `evalTaming`       | `'safeEval'`     | `'unsafeEval'` `'noEval'` | `eval` and `Function` of the start compartment. |
-| `stackFiltering`   | `'concise'`      | `'verbose'`    | deep stacks signal/noise   |
-| `overrideTaming`   | `'moderate'`     | `'min'` or `'severe'` | override mistake antidote  |
-| `overrideDebug`    | `[]`             | array of property names | detect override mistake |
-| `domainTaming`     | `'safe'`         | `'unsafe'`     | Node.js `domain` module |
-| `__hardenTaming__` | `'safe'`         | `'unsafe'`     | Making `harden` no-op for performance in trusted environments |
+| option                       | default setting  | other settings                         | about |
+|------------------------------|------------------|----------------------------------------|-------|
+| `regExpTaming`               | `'safe'`         | `'unsafe'`                             | `RegExp.prototype.compile` |
+| `localeTaming`               | `'safe'`         | `'unsafe'`                             | `toLocaleString`           |
+| `consoleTaming`              | `'safe'`         | `'unsafe'`                             | deep stacks                |
+| `errorTaming`                | `'safe'`         | `'unsafe'`                             | `errorInstance.stack`      |
+| `errorTrapping`              | `'platform'`     | `'exit'` `'abort'` `'report'` `'none'` | handling of uncaught exceptions |
+| `unhandledRejectionTrapping` | `'report'`       | `'none'`                               | handling of finalized unhandled rejections |
+| `evalTaming`                 | `'safeEval'`     | `'unsafeEval'` `'noEval'`              | `eval` and `Function` of the start compartment. |
+| `stackFiltering`             | `'concise'`      | `'verbose'`                            | deep stacks signal/noise   |
+| `overrideTaming`             | `'moderate'`     | `'min'` or `'severe'`                  | override mistake antidote  |
+| `overrideDebug`              | `[]`             | array of property names                | detect override mistake |
+| `domainTaming`               | `'safe'`         | `'unsafe'`                             | Node.js `domain` module |
+| `__hardenTaming__`           | `'safe'`         | `'unsafe'`                             | Making `harden` no-op for performance in trusted environments |
 
 In the absence of any of these options in lockdown arguments, lockdown will
-attempt to read these from the Node.js `process.env` environment.
-The corresponding environment variables are the same but in `UPPER_SNAKE_CASE`.
+attempt to read these options from the corresponding Node.js `process.env` environment.
```

Comment:
Might want to additionally explain, for those not familiar with this node feature, that for the start compartment these typically reflect *Nix environment variables in the context that the node process was launched from. Or just link to the @endo/env-options package.